pkgs/tools/security/creddump/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, python27, python27Packages }:
+
+python27Packages.buildPythonApplication rec {
+  pname = "creddump";
+  version = "0.3";
+  src = fetchFromGitHub {
+    owner = "moyix";
+    repo = "creddump";
+    rev = "ed95e1a9a920e04733a2950f7de0ff5bdfec631c";
+    sha256 = "1sy8r72x4c0qasg1p5wnxhfvymji2gvjx13k9qizag20fy80lcyi";
+  };
+
+  dontBuild = true;
```

Comment:
Why is that?

Reply:
`creddump` is a standalone python script, i.e., it comes without `setup.py`.
This leads to following error at build phase:
```
building
Executing setuptoolsBuildPhase
Traceback (most recent call last):
  File "nix_run_setup", line 8, in <module>
    exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\\r\\n', '\\n'), __file__, 'exec'))
IOError: [Errno 2] No such file or directory: 'setup.py'
```

That is why I omit the build phase

Reply:
:+1: Then perhaps:

```suggestion
  # No setup.py is available
  dontBuild = true;
```

Reply:
Then perhaps:

```suggestion
  # No setup.py is available
  dontBuild = true;
```

Reply:
Fixed

Reply:
You can mark this and other conversations as resolved in order to spare other reviewers from iterating this.